apps/betterangels-backend/shelters/migrations/0006_alter_shelter_options_shelter_reviewed.py

```diff
@@ -1,11 +1,8 @@
 from django.db import migrations, models
+from shelters.utils import assign_permissions_to_group_in_migration
 
 
 def create_shelter_data_entry_group(apps, schema_editor):
-    Group = apps.get_model("auth", "Group")
-    Permission = apps.get_model("auth", "Permission")
-    ContentType = apps.get_model("contenttypes", "ContentType")
```

Comment:
this code was being repeated in multiple migrations so pulled it out into a shelter util